e2e/test/scenarios/visualizations-charts/reproductions/21452-xhr-on-every-char-for-rename.cy.spec.js

```diff
@@ -36,8 +36,6 @@ describe("issue 21452", () => {
     cy.findByDisplayValue("Cumulative sum of Quantity").clear().type("Foo");
     // eslint-disable-next-line no-unscoped-text-selectors -- deprecated usage
     cy.findByText("Display type").click();
-    // Blur will result in another POST request which is expected
-    cy.wait("@dataset");
```

Comment:
New streamline logic works even before than before - there is no duplicate request when changing viz settings only.